### PR TITLE
New package: netsniff-0.6.5

### PR DESCRIPTION
**Pull request by bobertlo**
Disabled optional geoip support, because it does not seem to be maintained and does not work.

### CODE REVIEW

File: srcpkgs/libcli/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+pkgname=libcli
+version=1.9.8.4
+revision=1
+wrksrc=libcli-1.9.8-4
+build_style=gnu-makefile
+short_desc="Library for cisco style telnet interfaces"
+maintainer="Robert Lowry <bobertlo@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/dparrish/libcli"
+distfiles="https://github.com/dparrish/libcli/archive/v1.9.8-4.tar.gz"
```
Comment (Chocimier) [inline srcpkgs/libcli/template]: You can use auxiliary `_distver="${version%.*}-${version##*.}"`

File: srcpkgs/netsniff-ng/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'netsniff-ng'
+pkgname=netsniff-ng
+version=0.6.5
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --disable-geoip"
+hostmakedepends="pkg-config flex bison"
+makedepends="libcli-devel libnetfilter_conntrack-devel libpcap-devel
+	libnet-devel libsodium-devel liburcu-devel ncurses-devel zlib-devel"
```
Comment (CameronNemo) [inline srcpkgs/netsniff-ng/template]: Single space, not tab.

**Comment by bobertlo** [comment]
Thanks! On vacation, but will fix.

**Comment by bobertlo** [comment]
Should be good now.

File: srcpkgs/libcli/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'libcli'
+pkgname=libcli
+version=1.9.8.4
+_distver="${version%.*}-${version##*.}"
+revision=1
+wrksrc=libcli-1.9.8-4
```
Comment (Johnnynator) [inline srcpkgs/libcli/template]: You can use `_distver` here.

**Comment by bobertlo** [comment]
Oops. Fixed that.

**Pull request merged**